mono/metadata/appdomain.c

```diff
@@ -1412,6 +1416,21 @@ mono_domain_fire_assembly_load (MonoAssembly *assembly, gpointer user_data)
 	HANDLE_FUNCTION_RETURN ();
 }
 
+static gboolean
+mono_domain_asmctx_from_path (const char *fname, MonoAssembly *requesting_assembly, gpointer user_data, MonoAssemblyContextKind *out_asmctx)
+{
+	MonoDomain *domain = mono_domain_get ();
```

Comment:
It's usually good to always init out vars

Reply:
The hook invoke function `assembly_invoke_asmctx_from_path_hook` initializes the out var.  The individual hook functions only write to it when they return true.